package.json

```
@@ -197,7 +197,7 @@
         "package": "move-cli lingua-franca ../temp-lingua-franca89539275 && (vsce package; move-cli ../temp-lingua-franca89539275 lingua-franca)",
         "package-pre-release": "move-cli lingua-franca ../temp-lingua-franca89539275 && (vsce package --pre-release; move-cli ../temp-lingua-franca89539275 lingua-franca)",
         "deploy": "echo \"Y\r\n\" | code --install-extension vscode-lingua-franca-*.vsix --force",
-        "install": "npm run clean && npm run build && npm run package && npm run deploy",
+        "installExtension": "npm run clean && npm run build && npm run package && npm run deploy",
```

```suggestion
        "install-extension": "npm run clean && npm run build && npm run package && npm run deploy",
```
Because all other commands are snake case also.